docs/page.rst

```diff
@@ -347,6 +348,46 @@ In a nutshell, this is what you can do with PyMuPDF:
+
+      * New in version 1.23.0
+
+      Find tables on the page and return an object with related information. Typically, only very few of the many arguments ever need to be specified -- they mainly are tools to react to corner case situations.
+
+      :arg rect_like clip: specify a region to consider within the page rectangle. Default is the full page.
+      :arg list horizontal_lines: floats containing the y-coordinates of rows. If provided, there will be no attempt to identify additional table rows.
+      :arg list vertical_lines: floats containing the x-coordinates of columns. If provided, there will be no attempt to identify additional table columns.
+      :arg str vertical_strategy: request a search algorithm. The "lines" default looks for vector drawings. If "text" is specified, text positions are used to generate "virtual" column boundaries.
+      :arg str horizontal_strategy: request a search algorithm. The "lines" default looks for vector drawings. If "text" is specified, text positions are used to generate "virtual" row boundaries. The "text" choices are recommended when dealing with pages without any vector graphics -- like when this is an OCRed page.
+      :arg int min_words_vertical: to qualify as a table, at least this many rows must be present.
+      :arg int min_words_horizontal: to qualify as a table, at least this many columns must be present.
+
+      The remaining parameters are limits for merging different objects. For instance: Two horizontal lines with the same x-coordinates and a vertical distance less than 3 will be merged ("snapped") to one line.
+
+      :returns: a `TableFinder` object that has the following significant attributes:
+
+         * **cells:** a list of **all bboxes** on the page, that have been identified as table cells (across all tables). Each cell is a tuple `(x0, y0, x1, y1)` of coordinates or `None`.
+         * **tables:** a list of `Table` objects. This is `[]` if the page has no tables. Please note that while single tables can be found as items of this list, the `TableFinder` object itself is also a sequence of it tables. This means that if `tabs` is a `TableFinder` object, then table number `n` is delivered by `tabs.tables[n]` as well as by the shorter `tabs[n]`.
+
+
+         * Each `Table` object has the following attributes:
+
+            * **bbox:** the bounding box of the table as a tuple `(x0, y0, x1, y1)`.
+            * **cells:** bounding boxes of the table's cells (list of tuples). A cell may also be `None`.
+            * **extract:** a method returning the text content of each table cell (a list of list of strings).
```

Comment:
a list of list of ?

Reply:
I meant "list of lists of" - i.e. something in type annotation `List[List[String]]`. Not sure if we want to use this instead ...

Reply:
aha - gotcha - yes, I think the code sample would clarify it as a double nested array kind of thing.